src/details/ArborX_DetailsDistributedSearchTreeImpl.hpp

```diff
@@ -192,7 +192,7 @@ struct DistributedSearchTreeImpl
   template <typename ExecutionSpace, typename OutputView, typename Ranks,
             typename Distances = Kokkos::View<float *, DeviceType>>
   static void communicateResultsBack(MPI_Comm comm, ExecutionSpace const &space,
-                                     OutputView &view,
+                                     OutputView &out,
```

Comment:
What happened here? Hard to see from the diff.

Reply:
The name in the declaration and definition didn't match.